db/schema.rb

```diff
@@ -10,9 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_11_24_025600) do
+ActiveRecord::Schema[7.0].define(version: 2022_11_26_042109) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
Please add this line back in.

Reply:
Sorry not sure how it got removed. Have added back.